frontend/src/bundles/resume/store/slice.ts

```diff
@@ -36,6 +39,7 @@ const initialState: State = {
     resumeReview: null,
     templateSettings: null,
     currentTemplateId: null,
+    resumeViews: [],
```

Comment:
![image](https://github.com/BinaryStudioAcademy/bsa-winter-2023-2024-resumegemm/assets/24661681/fbc9fd5d-da43-493b-94c8-761846600b4f)

The `resumes` are fulfilled at the same moment as your `resumeViews`, can we just add the views logic to existing resumes? Or if not, you should get AWS image URLs, then fulfilling the resumes is redundant...